gitops/git/gitlab/gitlab.go

```diff
@@ -0,0 +1,68 @@
+package gitlab
+
+import (
+	"errors"
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/xanzy/go-gitlab"
+)
+
+var (
+	gitlabHost  = flag.String("gitlab_host", "https://gitlab.com", "The host name of the gitlab instance")
+	repo        = flag.String("gitlab_repo", "", "the repo to use for gitlab api requests")
+	accessToken = flag.String("gitlab_access_token", os.Getenv("GITLAB_TOKEN"), "the access token to authenticate requests")
+)
+
+func CreatePR(from, to, title string) error {
+	if *accessToken == "" {
+		return errors.New("github_access_token must be set")
```

Comment:
The message should be `"gitlab_access_token ... "`

Reply:
done